packages/crypto/src/lib/crypto.ts

```diff
@@ -184,7 +185,7 @@ export const combineEcdsaShares = async (
   const publicKey = Buffer.from(anyValidShare.publicKey, 'hex');
   const messageHash = Buffer.from(anyValidShare.dataSigned!, 'hex');
 
-  ecdsaVerify(variant!, messageHash, publicKey, [r, s, v]);
+  await ecdsaVerify(variant!, messageHash, publicKey, [r, s, v]);
```

Comment:
I'm pretty sure this had to be awaited

Reply:
I do not think we needed an await on this at the current point in time. but it shouldn't effect much since it will unblock in the same tick if the returned value is not a `Promise`